### PR TITLE
i18n(fr) Add french translation for `ghost.mdx`

### PR DESCRIPTION
**Pull request by McFlyPartages**
#### Description (required)
Add french translation for `ghost.mdx`

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/cms/ghost.mdx [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+});
+
+```
+
+
+### Afficher une liste d'articles
+
+La page `src/pages/index.astro` affichera une liste d'articles, chacun avec une description et un lien vers sa propre page.
+
+<FileTree title="Project Structure">
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/cms/ghost.mdx:128]:
```suggestion
<FileTree title="Structure du projet">
```

File: src/content/docs/fr/guides/cms/ghost.mdx [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+
+Cet exemple crée une page d'index qui répertorie les articles avec des liens vers des pages individuelles générées dynamiquement.
+
+### Récupération de données
+
+Vous pouvez récupérer les données de votre site avec l'API de contenu de Ghost.
+
+Tout d'abord, créez un fichier `ghost.ts` dans un répertoire `lib`.
+
+<FileTree title="Project Structure">
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/cms/ghost.mdx:98]:
```suggestion
<FileTree title="Structure du projet">
```

File: src/content/docs/fr/guides/cms/ghost.mdx [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+  readonly CONTENT_API_KEY: string;
+}
+```
+:::tip
+En savoir plus sur [l'utilisation des variables d'environnement](/fr/guides/environment-variables/) at les fichiers `.env` dans Astro.
+:::
+
+Votre répertoire racine doit maintenant contenir ces nouveaux fichiers :
+
+<FileTree title="Project Structure">
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/cms/ghost.mdx:51]:
```suggestion
<FileTree title="Structure du projet">
```

**Review by thomasbnt — CHANGES_REQUESTED** [review]
Some FileTree title are not translated, and a little word to change

**Review by thomasbnt — APPROVED** [review]
LGTM!

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work peeps 🙌

**Pull request merged**